### ISSUE TITLE
User can login at back-end if the password is wrong.

### ISSUE THREAD

**Issue by bhavikDigitalWebInfosoft** [joomla/joomla-cms#11869]
### Steps to reproduce the issue
- Do not give write permission to log file.
- Trying to login at the back-end using wrong password and correct username.
- There will be error message that, Cannot write to log file.
- At the top right corner, you can see user is logged-in in the site, which is not correct, because the password is wrong.
### Expected result

User can not access site with wrong password and stay at the login page.
### Actual result

Use can login with wrong password.
### Additional comments

![screen shot 2016-09-01 at 00 11 24](https://issues.joomla.org/uploads/1/ff0d2a777909f5c4269b05b0418e2dd5.png)

**Comment by hacki65** [comment]
I can confirm this. But this seems to be the normal Errorpage from Joomla! for these cases and it is confusing. But no User is really logged in and any action will bring back to the Loginpage.

**Comment by bertmert** [comment]
One could add a cosmetical
```
<?php if (!empty($user->id)) : ?>
 <ul class="nav nav-user<?php echo ($this->direction == 'rtl') ? ' pull-left' : ' pull-right'; ?>">
....
 </ul>
<?php endif; ?>
```
in templates/isis/error.php line 162.

**Comment by brianteeman** [comment]
This is a bug and needs fixing. The behaviour should be the same as if the logs file is writeable .

Thanks for reporting it

**Comment by mbabker** [comment]
The bug is just as @hacki65 pointed out, the error page is basically unaware of the user state.  Since the Isis error page tries to mimic the base template layout, it has to have additional checks to be aware of user state.  The Hathor error page doesn't do this.

**Comment by GeraintEdwards** [comment]
I am looking at this

**Comment by franz-wohlkoenig** [comment]
closed as having PR #16475<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11869">issues.joomla.org/tracker/joomla-cms/11869</a>.</sub>
